css/style.css

```
@@ -15,5 +15,147 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+ 
+ nav  {
+    display:flex;
+    justify-content: flex-end;
+     padding-top: 60px;
+     padding-left: 20px;
+     
+ } 
 
+.nav{
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 50%;
+    
+}
+#logo{
+   margin-right:39em;
+    
+    
+    
+}
+.nav-list {
+
+    display: flex;
+    margin-right: 5rem;
+  }
+  .nav-list a {
+    display:flex;
+    justify-content: flex-end;
+    display: block;
+    font-size: 1rem;
+    padding: 1rem;
+    text-decoration: none;
+    color: #2C3539;
+    
+  }
+  ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+  }
+  section{
+    background-image: url(/home/coder/GitHub/HTML-CSS-Coursework-Week2/img/first-background.jpg);
```

```suggestion
    background-image: url(../img/first-background.jpg);
```
We should use paths that are relative, so something like this. A relative path means I don't need to have the same computer as you `/home/coder/GitHub/HTML-CSS-Coursework-Week2/img/first-background.jpg`